Dockerfile

```diff
@@ -19,5 +18,14 @@ COPY . /usr/src/HoneyBadgerMPC
 
 RUN pip install -e pairing/
 
+# This is needed otherwise the build for the power sum solver will fail.
+# This is a known issue in the version of libflint-dev in apt.
+# https://github.com/wbhart/flint2/issues/217
+# This has been fixed if we pull the latest code from the repo. However, we want
+# to avoid compiling the lib from the source since it adds 20 minutes to the build.
+RUN sed -i '30c #include "flint/flint.h"' /usr/include/flint/flintxx/flint_classes.h
```

Comment:
ok good pragmatic save...
is there a way to make this assert that we're on the expected version of flint, would we notice if it that changed

Reply:
I have actually opened an issue on the flint repo - https://github.com/wbhart/flint2/issues/512

I will follow up with them regarding this.